WebHDFS.Kitty.IntegrationTests/Create.cs

```diff
@@ -10,10 +10,55 @@ public class Create
         private IWebHdfsClient client;
 
         [CheckConnStrSetupFact]
-        public async Task UploadFile()
+        public async Task UploadFileWithoutParams()
         {
             client = new WebHdfsClient(DataTestUtility.HdfsConnStr);
-            await client.UploadFile($"{DataTestUtility.HdfsRootDir}/sample", File.OpenRead("Samples/SampleTextFile.txt"));
+            await client.UploadFile($"{DataTestUtility.HdfsRootDir}/sample2", File.OpenRead("Samples/SampleTextFile.txt"));
+            var FileStat = await client.GetFileStatus($"{DataTestUtility.HdfsRootDir}/sample2");
+            Assert.True(FileStat.Type == "FILE");
+            Assert.True(FileStat.Permission == "755");
+        }
+
+        [CheckConnStrSetupFact]
+        public async Task UploadFileWithOverWrite()
+        {
+            client = new WebHdfsClient(DataTestUtility.HdfsConnStr);
+            var FileStat = await client.GetFileStatus($"{DataTestUtility.HdfsRootDir}/sample2");
+            var oldLenght = FileStat.Length;
+
+            await client.UploadFile($"{DataTestUtility.HdfsRootDir}/sample2", File.OpenRead("Samples/SampleTextFile2.txt"), Overwrite: true);
+            FileStat = await client.GetFileStatus($"{DataTestUtility.HdfsRootDir}/sample2");
+            Assert.True(FileStat.Length != oldLenght);
+        }
+
+        [CheckConnStrSetupFact]
+        public async Task UploadFileWith_No_OverWrite()
+        {
+            client = new WebHdfsClient(DataTestUtility.HdfsConnStr);
+            await client.Delete($"{DataTestUtility.HdfsRootDir}/sample2");

```

Comment:
Lets test it like:
1) Upload file
2) Check file
3) Upload file
4) Catch exception